src/manifest.json

```
@@ -29,5 +29,11 @@
   "permissions": [
     "contextMenus",
     "storage"
-  ]
+  ],
+  "commands": {
+    "_execute_action": {},
+    "shoot": {
+      "description": "Take a screenshot"
+    }
```

デフォルト設定だけ一応追加お願いしますー！:pray:
今まで設定されていたキー設定をデフォで置いて貰えると:pray:

```suggestion
    "shoot": {
      "description": "Take a screenshot",
      "suggested_key": {
        "default": "Ctrl+Q",
        "mac": "MacCtrl+Q"
      }
    }
```